packages/js/product-editor/webpack.config.js

```diff
@@ -43,7 +43,6 @@ module.exports = {
 					? `[name].css`
 					: `[name]/style.css`;
 			},
-			chunkFilename: 'chunks/[id].style.css',
```

Comment:
This is not needed for the same reason as in the `ai` package. This webpack config builds only styles and blocks in `src/blocks`. None of them uses any dynamic chunks.